src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.ts

```diff
@@ -47,6 +52,8 @@ export class BuilderNavbarComponent implements OnDestroy {
     private toasterService: ToasterService,
     private collectionService: CollectionService,
     private contextMenuService: ContextMenuService,
+    private changelogService: ChangelogService,
+    private object: LearningObjectService,
```

Comment:
This is not a descriptive variable name. I only noticed because I was very thrown off by the code below. The name should identify that it is a service, so that developers understand that it is a datasource that uses asynchronous, network-based operations.